sdk/formrecognizer/ai-form-recognizer/CHANGELOG.md

```diff
@@ -1,7 +1,8 @@
 # Release History
 
-## 1.0.0-preview.2 (Unreleased)
+## 1.0.0-preview.2 (2020-05-05)
 
+`FormTrainingClient.listModels()` now works correctly on NodeJs v8.
```

Comment:
total nitpick: we usually do list formatting in the changelog. I would just add a `- ` at the beginning for consistency

Reply:
Will do